README.md

```diff
@@ -40,29 +28,75 @@ In the source folder of your application, for Nano S:
 
 ```bash
 $ # docker can be replaced with podman or buildah without sudo
-$ sudo docker run --rm -ti -v "$(realpath .):/app" ledger-app-builder:latest
+$ sudo docker run --rm -ti -v "$(realpath .):/app" --user $(id -u $USER):$(id -g $USER) ledger-app-builder:latest
```

Comment:
I'm not sure, should it be either:
- ledger-app-builder:latest (manual build)
- ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder:latest (image from ghcr)

Reply:
`ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder:latest` is better, I'll change.